jina/resources/logging.default.yml

```diff
@@ -1,5 +1,5 @@
 handlers:  # enabled handlers, order does not matter
-  - StreamHandler
```

Comment:
what is this change, what does it imply? 

Can u add both Handlers as default?

Reply:
The change is now that the main logger is the Rich logger. If we had both, the log are going to be print two times, one with rich one with the StreamHandler